src/cloud-api-adaptor/docker/prereqs.sh

```diff
@@ -86,7 +86,7 @@ install_os_packages() {
 # the function will download the packages, extract them and install them in /usr/local/bin
 # Following are the packages that are installed:
 # yq=https://github.com/mikefarah/yq/releases/download/v4.44.2/yq_linux_amd64
-# kubectl=https://storage.googleapis.com/kubernetes-release/release/v1.27.11/bin/linux/amd64/kubectl
+# kubectl=https://storage.googleapis.com/kubernetes-release/release/v1.29.4/bin/linux/amd64/kubectl
```

Comment:
Hi @bpradipt ,

am I missing something or this just changed the comment line? Probably you want to update the version listed in `REQUIRED_BINARY_PACKAGES`, right?

Reply:
yes ;) .. Nice catch